modules/virt-vm-storage-volume-types.adoc

```diff
@@ -55,6 +54,14 @@ require persistent storage.
 Only RAW and QCOW2 formats are supported disk types for the container
 image registry. QCOW2 is recommended for reduced image size.
 
+[NOTE]
+====
+A containerDisk volume is ephemeral. It is discarded when
```

Comment:
Ooh, also, we aren't supposed to hard wrap anymore :(

Reply:
In-flight PR! In-flight PR! 
But unwrapped the whole table anyway :)